rpcd/playbooks/verify-maas.yml

```diff
@@ -23,7 +23,7 @@
         . {{ maas_venv_bin }}/activate
         {% endif %}
         {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }} \
+        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
```

Comment:
{{ maas_fqdn_extension }} is set in rpcd/etc/openstack_deploy/user_rpco_variables_defaults.yml , but by default it's empty. So I believe it makes no difference only if user or tester gives customized FQDN. That means we need to define FQDN then see what happens/verify this.